datahub-kubernetes/datahub/templates/datahub-upgrade/datahub-restore-indices-job-template.yml

```diff
@@ -0,0 +1,119 @@
+{{- if .Values.datahubUpgrade.enabled -}}
```

Comment:
Is this correct? Why does this depend on datahubUpgrade flag?

Reply:
All k8s files under datahub-upgrade are all enabled by the same flag

Reply:
Enabling it doesn't do much until you actually run the job since these cronjobs are suspended by default